state/auth_export/accounts.json

```diff
@@ -1 +1 @@
-{"kind":"identitytoolkit#DownloadAccountResponse","users":[{"localId":"V2gb6iTXVnjHWd1jSmnGNuJc8rNg","lastLoginAt":"1686082007919","emailVerified":false,"email":"test@test.com","salt":"fakeSaltogTkLs67l1S2j8GZKqru","passwordHash":"fakeHash:salt=fakeSaltogTkLs67l1S2j8GZKqru:password=iamatest","passwordUpdatedAt":1686080130786,"validSince":"1686080130","createdAt":"1686080130785","providerUserInfo":[{"providerId":"password","email":"test@test.com","federatedId":"test@test.com","rawId":"test@test.com"}],"lastRefreshAt":"2023-06-06T22:58:20.530Z"}]}
\ No newline at end of file
+{"kind":"identitytoolkit#DownloadAccountResponse","users":[{"localId":"euAEb7ntPfXt1w3u1mUiYqT72XFd","lastLoginAt":"1686865276387","emailVerified":false,"email":"test@test.com","salt":"fakeSaltbYjD03UkE4KigpHIGr8E","passwordHash":"fakeHash:salt=fakeSaltbYjD03UkE4KigpHIGr8E:password=iamatest","passwordUpdatedAt":1686862046337,"validSince":"1686862046","createdAt":"1686862046337","providerUserInfo":[{"providerId":"password","email":"test@test.com","federatedId":"test@test.com","rawId":"test@test.com"}],"lastRefreshAt":"2023-06-15T21:41:16.387Z"}]}
```

Comment:
This looks like test data but just want to confirm, are these credentials live anywhere?

Reply:
Nope, just emulator state, completely safe to publish this.